src/Instrumentation/Slim/_register.php

```diff
@@ -3,6 +3,11 @@
 declare(strict_types=1);
 
 use OpenTelemetry\Contrib\Instrumentation\Slim\SlimInstrumentation;
+use OpenTelemetry\SDK\Sdk;
+
+if (Sdk::isInstrumentationDisabled('slim') === true) {
```

Comment:
While I only had it with Slim, is it an idea to add `-framework` to this one (and the others?) this is fine too with me, but just as a general note

Reply:
We have to document that somewhere. I'm thinking whether we can verify values passed via `OTEL_PHP_DISABLED_INSTRUMENTATION`, it's easy to make mistake

Reply:
Agreed that it will need to be documented, we should probably list them over in our docs in opentelemetry.io
Rather than using string values such as `slim`, I'd like to see constants, eg `SlimInstrumentation::NAME`.
re: naming, I think that the major frameworks are well-known enough that "slim", "laravel", "symfony" etc will be okay

Reply:
Alright, so if I understand correctly, you want the names to be added to their respective `*Instrumentation`-classes and then used in these places. 

I at first thought that @pdelewski meant that we needed to add these to the main repo like we have for other OTEL vars. Which I wasn't totally sure about. 

I'll update the repo to add the names in public constants and use them here. I'll open a docs PR later (if thats possible and if I find the location of them :D )

Reply:
I was also thinking about using constants instead of strings